hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerRatisServer.java

```diff
@@ -184,9 +183,17 @@ private OMResponse processReply(OMRequest omRequest, RaftClientReply reply)
         OMResponse.Builder omResponse = OMResponse.newBuilder();
         omResponse.setCmdType(omRequest.getCmdType());
         omResponse.setSuccess(false);
-        omResponse.setMessage(stateMachineException.getCause().getMessage());
-        omResponse.setStatus(parseErrorStatus(
-            stateMachineException.getCause().getMessage()));
+        if (stateMachineException.getCause() != null) {
+          omResponse.setMessage(stateMachineException.getCause().getMessage());
+          omResponse.setStatus(
+              exceptionToResponseStatus(stateMachineException.getCause()));
+        } else {
+          // Current Ratis is setting cause, this is an safer side check.
+          LOG.error("StateMachine exception cause is not set");
+          omResponse.setStatus(
```

Comment:
We should set the error message also here, right? Might break something else to not have a error message.

Reply:
The downstream client only checks status. As here cause is null, so skipped the setMessage.

If for sure, we want to set error message, I see 2 options.
1. 
if (stateMachineException.getMessage() != null) 
omResponse.setMessage(stateMachineException.getMessage())

2.
 omResponse.setMessage(org.apache.hadoop.util.StringUtils.stringifyException(stateMachineException);

Let me know what you prefer?

Reply:
I am ok with either.

Reply:
Done. Used 2nd approach.